src/components/Layout.re

```diff
@@ -93,7 +93,7 @@ module Header = {
                      <span className=Styles.someMarginRight>
                        <S>
                          {usedtranslationModeContext->translationModeCrypto
-                            ? "EXPERT MODE" : "SIMPLE MODE"}
+                            ? "FULL NERD MODE" : "SIMPLE MODE"}
```

Comment:
Hmm, I prefered "expert" here potentially. Also  that is the default mode. 

I'm also really nervous about the affordance of that switch. Will people who aren't us (the creators) ever think to click it? https://uxplanet.org/ux-design-glossary-how-to-use-affordances-in-user-interfaces-393c8e9686e4

Reply:
On wording, I just thought it was maybe more playful, but yeah much of a muchness. I agree, we should consider simple mode as default I believe, then prompt the nerds to go to expert mode. People will be stubborn to go to simple mode, but to go to expert mode they will take it as a challenge. 

Is the word simple a bit demeaning? Perhaps, "plain english", "No buzzwords", "regular mode" etc... ?

Reply:
I think I prefer Expert, but if you both want nerd then rather make it "NERD MODE" and remove the "full"

Reply:
Also should we make Simple mode the default when people open the site.

Reply:
Happy with expert. Agree on the default.